Makefile

```diff
@@ -429,6 +429,12 @@ ifndef DISABLE_JEMALLOC
 	PLATFORM_CCFLAGS += $(JEMALLOC_INCLUDE)
 endif
 
+DISABLE_PERF_CONTEXT ?= 0
```

Comment:
Shouldn't this be `1`? by default?

Reply:
@rkhachatryan
 I initially set it to 1, but since there are some UTs depending on perf_context, so it may cause CI to fail.
So the way I used is to set the default value to 0 in the `Makefile` and then set it to 1 in the build script of frocksdb , it's that ok ?

Reply:
Thanks for clarifying, I'd be fine with that, but I don't see this change in the PR:
> and then set it to 1 in the build script of frocksdb
Or am I missing something?

Reply:
@rkhachatryan I added` DISABLE_PERF_CONTEXT=1` in the compilation script for Mac (`FROCKSDB_VERSION=1.0 PORTABLE=1 ROCKSDB-DISMABLE_JEMALLOC=true DEBUG_LEVEL=0`) to take effect in Mac products, and then added a commit to take effect in Docker compiled products (changes in `docker-build-linux-centos.sh` and `docker-build-linux-alpine.sh`)

Reply:
> @rkhachatryan I added DISABLE_PERF_CONTEXT=1 in the compilation script for Mac (FROCKSDB_VERSION=1.0 PORTABLE=1 ROCKSDB-DISMABLE_JEMALLOC=true DEBUG_LEVEL=0) 

Did you add it in this PR? I couldn't find this change.

> added a commit to take effect in Docker compiled products (changes in docker-build-linux-centos.sh and docker-build-linux-alpine.sh)

Should we also update non-Docker scripts?

Reply:
@Zakelly I would like to ask,if I want to add DISABLE_PERF_CONTEXT=1 to the compilation script of all frocksdb environments. What do you think about this, or do you have any better suggestions?

Reply:
> @Zakelly I would like to ask,if I want to add DISABLE_PERF_CONTEXT=1 to the compilation script of all frocksdb environments. What do you think about this, or do you have any better suggestions?

I'd be OK with this.